client/src/components/snippets/sidebar/FileTypeSidebar.tsx

```
@@ -0,0 +1,145 @@
+      {/* Filters */}
+      <div className="flex-1 overflow-y-auto">
+        <div className="p-2">
+          {/* All Files */}
+          <button
+            onClick={() => onSelectType(null)}
+            className={`w-full flex items-center justify-between px-3 py-2 rounded-lg transition-colors ${
+              selectedType === null
+                ? "bg-light-primary dark:bg-dark-primary text-white"
+                : "hover:bg-light-hover dark:hover:bg-dark-hover text-light-text dark:text-dark-text"
+            }`}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)